addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -65,11 +72,19 @@ const ThreadPatch = {
                 }
             },
         });
-    },
-    get videoCount() {
-        return Object.values(this.store["discuss.channel.rtc.session"].records).filter(
-            (session) => session.hasVideo
-        ).length;
+        this.videoCount = fields.Attr(0, {
+            compute() {
+                return Object.values(this.store["discuss.channel.rtc.session"].records).filter(
+                    (session) => session.hasVideo
+                ).length;
+            },
+            onUpdate() {
+                const shouldShowIndicator = this.videoCount > 0  &&
+                      this.chat_window?.isOpen &&
+                      this.fullscreenIndicator !== INDICATOR.DISCARDED;
+                this.fullscreenIndicator = shouldShowIndicator ? INDICATOR.ACTIVE : INDICATOR.DISABLED;
```

Comment:
If the indicator was discarded, it should stay discarded, rather than becomes `DISABLED`, otherwise next onUpdate will show the pulse again.